app/_lib/radar/rest-source/service.ts

```diff
@@ -1,16 +1,28 @@
+import { URLSearchParams } from "next/dist/compiled/@edge-runtime/primitives/url"
+
 const AUTH_BASE_URL = process.env.NEXT_PUBLIC_HYDRA_PUBLIC_URL + "/oauth2"
 const CLIENT_ID = process.env.SEP_CLIENT_ID ?? "SEP"
 const REDIRECT_URI = process.env.NEXT_PUBLIC_SEP_REDIRECT_URI
+const AUDIENCE = "res_restAuthorizer"
```

Comment:
I noticed we are using this for the SEP token, do we need to add the other resources as well like gateway or is this mostly going to be used for rest-source-auth?

Reply:
We should probably add other resources since this token can be used for all client-side requests to other parts of RADAR. Right now that is just Rest Auth, but presumably other things might be useful in the future?

Reply:
Oh ok yes we can just keep rest auth for now.